lib/linguist/languages.yml

```diff
@@ -4496,6 +4496,13 @@ Slash:
   tm_scope: text.html.slash
   ace_mode: text
   language_id: 349
+Slice:
+  type: programming
+  tm_scope: source.slice
+  ace_mode: text
+  extensions:
+  - ".ice"
+  language_id: 894641667
```

Comment:
Errm, don't you want a colour associated with this language? At the moment it's going to fallback to the default of `#cccccc`.

Looking at https://doc.zeroc.com/ice/latest/the-slice-language, it seems blue is the colour of choice so it's probably going to be a PITA to find one that doesn't conflict with one already associated with a language on GitHub.com - blue is really really popular.

Reply:
Whoops, would definitely like a colour. Can we use `#477ed0`?

Reply:
I doubt it. Various shades of blue are already highly used. Try it. Add the `color:` entry and then run `bundle exec ruby test/test_color_proximity.rb` (I'm assuming you've already run `script/bootstrap`). Rinse and repeat until you find a colour the test likes, then commit your colour and push.

Reply:
Thanks @lildude, I found a blue! `#003fa2`

Reply:
🎉 nice!!